### DISCUSSION TITLE
AMD Ryzen Vega GPU

### DISCUSSION THREAD

**Discussion by veno77** [blakeblackshear/frigate#2126] [Ask A Question]
Any ideas how can I enable the hardware acceleration of AMD Vega IGPU ?

**Comment by Yonny24** [comment]
Following. I have another pc with a ryzen 5 3400g. Tried Frigate on that as a trial (in an ubuntu docker container), just used all the CPU no matter what arguments I passed in. I don't think gpu's can be used.

**Reply by peter247** [reply]
I think I may have got hardware acceleration at leased part working on my ryzen 5700g with :

`cameras:
  driveway:
    ffmpeg:
      hwaccel_args:
        - -hwaccel
        -  vaapi
        - -hwaccel_device
        - /dev/dri/renderD128
      inputs:
        - path: rtsp://admi`
and removing i965 from dockers env var and using radeonsi

using top ffmpeg went from 33% to 21% .
But was trying to use acceleration to record , but ended up with a green screen , and found just removing what I ended and restarting docker doesn't fix it , I had to reboot the machine also reboot the camera to get it working again .
